R/plumber.R

```diff
@@ -177,10 +182,17 @@ plumber <- R6Class(
       private$notFoundHandler <- default404Handler
 
       if (!is.null(file)){
-        private$lines <- readLines(file)
-        private$parsed <- parse(file, keep.source=TRUE)
+        con <- file(file, encoding=utf8Encoding)
+        on.exit(close(con), add=TRUE)
+
+        private$lines <- readLines(con)
+        Encoding(private$lines) <- utf8Encoding
```

Comment:
I believe this line is not needed as the `con` already has the encoding put in.

Reply:
```
> b <- url("https://raw.githubusercontent.com/trestletech/plumber/b8cbe3afd0b4a5d8f012f4e06e5248e45535e785/tests/testthat/files/multibytes.R", encoding = "UTF-8")
> readLines(b)
[1] "#* @get /echo"  "function() {"   "  \"中文消息\"" "}"     
```

Reply:
You are right about this.